cluster/cloudimages/k8s-ebs-jessie-amd64-hvm.yml

```diff
@@ -0,0 +1,66 @@
+---
+name: k8s-1.2-debian-{system.release}-{system.architecture}-{provider.virtualization}-{%Y}-{%m}-{%d}-ebs
+provider:
+  name: ec2
+  virtualization: hvm
+  enhanced_networking: simple
+  description: Kubernetes 1.2 Base Image - Debian {system.release} {system.architecture}
+bootstrapper:
+  workspace: /target
+  # tarball speeds up development, but for prod builds we want to be 100% sure...
+  # tarball: true
+system:
+  release: jessie
+  architecture: amd64
+  # We use grub, not extlinux.
+  # See https://github.com/andsens/bootstrap-vz/issues/182
+  # extlinux makes it harder to modify boot args, and may have reboot problems
+  # bootloader: extlinux
+  bootloader: grub
+  charmap: UTF-8
+  locale: en_US
+  timezone: UTC
+volume:
+  backing: ebs
+  partitions:
+    type: msdos
+    root:
+      filesystem: ext4
+      size: 8GiB
+packages:
+  mirror: http://cloudfront.debian.net/debian
+  install:
+    # cloud-initramfs-growroot will resize the master partition on boot
+    - cloud-initramfs-growroot
+    # These packages would otherwise be installed during first boot
+    - aufs-tools
+    - curl
+    - python-yaml
+    - git
+    - nfs-common
+    - bridge-utils
+    - logrotate
+    - socat
+    - python-apt
+    - apt-transport-https
+    - unattended-upgrades
+    - lvm2
+    - btrfs-tools
+plugins:
+  cloud_init:
+    metadata_sources: Ec2
+    username: admin
+  ntp: {}
```

Comment:
The empty block here means "install ntp and don't configure any nameservers".  As it stands, this is actually equivalent to just adding ntp to the package list.

The default NTP servers on debian jessie are:

```
server 0.debian.pool.ntp.org iburst
server 1.debian.pool.ntp.org iburst
server 2.debian.pool.ntp.org iburst
server 3.debian.pool.ntp.org iburst
```

AWS does have NTP servers, but they are also just drawn from the public NTP servers.  Documented here: http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/set-time.html

> Note: The n.amazon.pool.ntp.org DNS records are intended to load balance NTP traffic from AWS. However, these are public NTP servers in the pool.ntp.org project, and they are not owned or managed by AWS. There is no guarantee that they are geographically located near your instances, or even within the AWS network. For more information, see http://www.pool.ntp.org/en/.

I kept the empty ntp block because I figure (1) the plugin may do more in future and (2) it'll be clearer if we have a GCE image which will have a useful configuration block.